apps/passport-client/src/localstorage.ts

```diff
@@ -20,19 +21,20 @@ export async function loadPCDs() {
 
   await SemaphoreGroupPCDPackage.init({
     wasmFilePath: SERVER_STATIC_URL + "/semaphore-artifacts/16.wasm",
-    zkeyFilePath: SERVER_STATIC_URL + "/semaphore-artifacts/16.zkey",
+    zkeyFilePath: SERVER_STATIC_URL + "/semaphore-artifacts/16.zkey"
   });
 
   await SemaphoreSignaturePCDPackage.init({
     wasmFilePath: SERVER_STATIC_URL + "/semaphore-artifacts/16.wasm",
-    zkeyFilePath: SERVER_STATIC_URL + "/semaphore-artifacts/16.zkey",
+    zkeyFilePath: SERVER_STATIC_URL + "/semaphore-artifacts/16.zkey"
   });
 
   return await PCDCollection.deserialize(
     [
       SemaphoreGroupPCDPackage,
       SemaphoreIdentityPCDPackage,
       SemaphoreSignaturePCDPackage,
+      JubJubSignaturePCDPackage
```

Comment:
Done, [updated PCD package](https://github.com/dmpierre/jubjub-signature-pcd/commit/7192bfc662989edae532eedbb47423c34533deb9) to avoid `SyntaxError: Unexpected token export`.